contracts/eosio.system/eosio.system.cpp

```diff
@@ -11,6 +11,6 @@ extern "C" {
 
     /// The apply method implements the dispatch of events to this contract
     void apply( uint64_t code, uint64_t act ) {
-       eosiosystem::contract<N(eosio)>::apply( code, act );
+//       eosiosystem::contract<N(eosio)>::apply( code, act );
```

Comment:
`eosio_run_test.py` is expecting this behavior to work, is there a reason why the `eosio.system` contract is currently disabled?